README.md

```diff
@@ -1 +1,36 @@
-# cubes_traveller
\ No newline at end of file
+# cubes_traveller
+
+## File Structure
+Example:
+- Assets
+    - Audio
+    - Scripts
+        - Player
+        - ...
+    - Models
+        - Player
+        - Cube
+        - ...
+    - Materials
+    - Prefabs
+        - Interactables
+    - Scenes
+    - Textures
+        - Player
+        - ...
+    - Shaders
+        - ...
+
+## Naming Conventions
+### Variables
+camelCase
+### Functions, Methods & Classes
+PascalCase
```

Comment:
It could be interesting to separate :
- function arguments (I usually prefix them with '_') 
- struct/class members (can be prexied with 'm_')
- function local variables

like this :
```

struct MaterialSample
{
    float3 m_albedo;
    float3 m_normal;
    ...
};
float3 GetNormal(in float _scale, in MaterialSample _material)
{
    float3 normal = _material.m_normal;
    normal *= _scale;

    return normal;
}
```